### ISSUE TITLE
when stop/start forget the starting mode (-g, -s, etc)

### ISSUE THREAD

**Issue by joseblas** [LukeTillman/dse-docker#12]
Given a container with -g -s (Graph and Solr), 
When you stop it (docker stop)
When you restart it (docker start)
Then DSE is in Cassandra mode (no Solr neither Graph)

 Regards, J

**Comment by LukeTillman** [comment]
I can't seem to reproduce this. Here are the steps I tried.

1. Start DSE in search mode
```
> docker run --name my-dse -d luketillman/datastax-enterprise:5.0.5 -s
```
2. Checked the logs (`docker logs my-dse`) and found this line indicating it had started in search mode:
```
INFO  22:08:19,542  DseDelegateSnitch.java:44 - Setting my workload to Search
```
3. Stopped the node
```
> docker stop my-dse
```
4. Started the stopped container.
```
> docker start my-dse
```
5. Checked the logs again and found another line indicating it had started in search mode again:
```
INFO  22:11:36,040  DseDelegateSnitch.java:44 - Setting my workload to Search
```

As far as I can tell, Docker is supposed to remember all the arguments passed to a container on `docker run` so it can reuse them if the container is stopped and then started again. I'm going to close this for now, but if you have any further details feel free to comment and I can try again. Thanks!
